tests/posix/fs/testcase.yaml

```diff
@@ -3,6 +3,7 @@ common:
   arch_exclude:
     - nios2
   platform_exclude:
+    - esp32s3_touch_lcd_1_28/esp32s3/appcpu
```

Comment:
What was the reason for this exclusion?

Reply:
@cfriedt  RAM overflowed in this target.

Reply:
Would be best to add a line like this:

```yaml
min_ram: 16
```

And then we would not need to filter by platform, so the specific exclusion can be removed.

See https://docs.zephyrproject.org/latest/develop/test/twister.html